typescript/sdk/src/consts/chainConnectionConfigs.ts

```diff
@@ -103,6 +103,36 @@ export const goerli: IChainConnection = {
   apiPrefix: 'api-',
 };
 
+export const optimismgoerli: IChainConnection = {
+  provider: new ethers.providers.JsonRpcProvider(
+    'https://goerli.optimism.io',
+    420,
+  ),
+  confirmations: 1,
+  blockExplorerUrl: 'https://goerli-optimism.etherscan.io/',
+  apiPrefix: 'api-',
+};
+
+export const arbitrumgoerli: IChainConnection = {
+  provider: new ethers.providers.JsonRpcProvider(
+    'https://goerli-rollup.arbitrum.io/rpc	',
+    421613,
+  ),
+  confirmations: 1,
+  blockExplorerUrl: 'https://goerli.arbiscan.io',
+  apiPrefix: 'api-',
+};
+
+export const zksync2testnet: IChainConnection = {
```

Comment:
Did we get zksync working? if not, inclusion around the SDK may cause some confusion. Not a blocking issue though if support is imminent

Reply:
Yeah per PR description, I'm likely going to reattmpt it soon